operator/pkg/central/values/translation/translation.go

```diff
@@ -219,15 +173,6 @@ func getCentralComponentValues(c *platform.CentralComponentSpec, checker *pvcSta
 	cv.SetStringMap("nodeSelector", c.NodeSelector)
 	cv.AddAllFrom(translation.GetTolerations(translation.TolerationsKey, c.Tolerations))
 
-	// TODO(ROX-7147): design CentralEndpointSpec, see central_types.go
```

Comment:
Why did you remove this TODO?

Reply:
ROX-7147 is closed. This comment do not seem to directly relate to the contexts. Do you want to keep it longer?

Reply:
So seems fine to remove it, just wanted to make sure it wasn't accidental.